transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketChannelTest.java

```diff
@@ -355,12 +380,12 @@ public void channelReadComplete(ChannelHandlerContext ctx) throws Exception {
 
         void assertSingleRead() throws InterruptedException {
             assertTrue(latch.await(5, TimeUnit.SECONDS));
-            assertEquals(1, count.get());
+            assertTrue("count: " + count.get(), count.get() >= 1);
```

Comment:
@normanmaurer - I see what you are saying ... the only "reasonable" option is `0` if the assertion fails because of the conditional so we should be able to infer the value even if we remove the string ... let me just change it